src/goLanguageServer.ts

```diff
@@ -376,8 +376,8 @@ function registerUsualProviders(ctx: vscode.ExtensionContext) {
 	vscode.workspace.onDidChangeTextDocument(parseLiveFile, null, ctx.subscriptions);
 }
 
-const defaultLatestVersion = semver.coerce('0.1.7');
-const defaultLatestVersionTime = moment('2019-09-18', 'YYYY-MM-DD');
+const defaultLatestVersion = semver.coerce('0.3.1');
+const defaultLatestVersionTime = moment('2020-02-04', 'YYYY-MM-DD');
```

Comment:
@ramya-rao-a: Can we get away from having this hardcoded default as a backup?

Reply:
We had this for the case when user doesnt want the extension to keep checking with the proxy server to determine if an update is needed. Removing the default would mean that such users will never get the prompt to update `gopls`